rllib/evaluation/collectors/agent_collector.py

```diff
@@ -88,9 +89,10 @@ def __init__(
         self.max_seq_len = max_seq_len
         self.disable_action_flattening = disable_action_flattening
         self.view_requirements = view_reqs
-        self.intial_states = intial_states or []
+        self.intial_states = intial_states if intial_states is not None else []
```

Comment:
intial_states can be an np array, in which case this throws an error about not being convertible to boolean.